README.md

```diff
@@ -8,7 +8,7 @@ This image is based on [Alpine Linux](https://alpinelinux.org/) to be lightweigh
 
 To use it try running: 
 ```
-curl 'https://api.github.com/repos/stedolan/jq/commits?per_page=1' | docker run -i --rm  euranova/jq[optional-tag] jq .[0].commit
+curl 'https://api.github.com/repos/stedolan/jq/commits?per_page=1' | docker run -i --rm  euranova/jq jq .[0].commit
```

Comment:
I would have two different bits of code. One with the explanation of howit works, and *then* an example, clearly stated as such. Use brackets or something for the bits that vary, such as `[process outputing json] | bla bla`.